appservice/src/createAppService/SiteCreateStep.ts

```diff
@@ -105,6 +100,12 @@ export class SiteCreateStep extends AzureWizardExecuteStep<IAppServiceWizardCont
     }
 }
 
+export function getNewFileShareName(siteName: string): string {
+    const randomLetters: number = 6;
```

Comment:
Why 6?  Is this how the portal does it?

Reply:
Because 6 is my favorite number lol :) I think they might do 4